KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/RecordingModule.java

```diff
@@ -237,7 +237,9 @@ public void onConnectAccept(IClient client)//Should called after onConnect (if a
             // but the key to get liveEntry from entry's persistent data is available
             String entryId = Utils.getEntryIdFromClient(client);
             KalturaLiveEntry liveEntry = (KalturaLiveEntry)KalturaEntryDataPersistence.getPropertyByEntry(entryId, Constants.CLIENT_PROPERTY_KALTURA_LIVE_ENTRY);
-            if (liveEntry.recordStatus == null || liveEntry.recordStatus == KalturaRecordStatus.DISABLED){
+
+            boolean isNewLiveRecordingEnabled = KalturaAPI.getKalturaAPI().isNewRecordingEnabled(liveEntry);
+            if (!isNewLiveRecordingEnabled || liveEntry.recordStatus == null || liveEntry.recordStatus == KalturaRecordStatus.DISABLED){
                 return;
```

Comment:
https://github.com/kaltura/media-server/blob/da1786ac8f2c93598034115c5f728dfe5cf1b9c4/KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/RecordingModule.java#L372 . can be removed now...